packages/react-icons/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "spnels-fluent-system-test0000",
+  "version": "1.1.50",
+  "description": "Fluent System Icons are a collection of familiar, friendly, and modern icons from Microsoft.",
+  "author": "Microsoft Inc.",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui-system-icons.git"
+  },
+  "scripts": {
+    "clean": "shx rm -rf ./components",
+    "cleanSvg": "shx rm -rf ./intermediate",
+    "copy": "node ../../importer/generate.js --source=../../assets --dest=./intermediate --extension=svg --target=react",
```

Comment:
I didn't get a chance to experiment with this, but I was thinking the workflow should be to consume the optimized icons from the svg package. That way all logic about optimizing goes there and we can make any future updates in one place.

Reply:
Doing that will require that the SVG package be given any treatments needed for React. Are there any possible uses or properties that the React package will need that would be blocked by the optimizations for the SVG package? That's not my expertise, so I'll defer to others with more experience, but my first concern is the styling options, particularly as color is concerned. We have been asked to address possible multi-color icon scenarios. The way I see for that is to assign a color to individual paths, but all fills have been removed from the SVGs at the moment.

Reply:
It seems we'd need/want to support the same color capabilities across all published formats, so I don't see any reason React would need something that we were removing in svg. If anything it allows us to standardize the way we treat the icons, for instance anywhere we support a secondary color or fill it would work the same way at the actual svg level. The interface might vary based on implementation.

Reply:
Given that we don't want to block this, I'll add an issue to handle multi-color options and specifically mention tightening up on this flow.